samples/highcharts/data-tools/datacursor-html/demo.js

```diff
@@ -0,0 +1,120 @@
+const container1 = document.getElementById('container1');
+const container2 = document.getElementById('container2');
+const sort = new Highcharts.DataModifier.types.Sort({
+    direction: 'asc',
+    orderByColumn: 'City'
+});
+const table = new Highcharts.DataTable({
+    Rank: [1, 2, 3, 4, 5, 6],
+    City: ['Sydney', 'Melbourne', 'Brisbane', 'Perth', 'Adelaide', 'Canberra'],
+    State: ['NSW', 'VIC', 'QLD', 'WA', 'SA', 'ACT'],
+    Population: [5259764, 4976157, 2568927, 2192229, 1402393, 453558]
+});
+const tableModified = sort.modifyTable(table.clone());
+
+
+// Render HTML Tables
+
+function renderTable(container, table) {
+    const columnNames = table.getColumnNames();
+    const html = [];
+
+    html.push('<table>');
+    // Render column header
+    html.push('<thead>');
+    html.push('<tr>');
+    for (const columnName of columnNames) {
+        html.push(`<th>${columnName}</th>`);
+    }
+    html.push('</tr>');
+    html.push('</thead>');
+    // Render row body
+    html.push('<tbody>');
+    for (let j = 0, jEnd = table.getRowCount(), row; j < jEnd; ++j) {
+        row = table.getRow(j);
+        // We use the Rank column to identify each row
+        html.push(`<tr data-rank="${row[0]}">`);
+        for (let i = 0, iEnd = row.length; i < iEnd; ++i) {
+            html.push(
+                // We use the column name to identify a cell
+                `<td data-column="${columnNames[i]}">`,
+                `${row[i]}`,
+                '</td>'
+            );
+        }
+        html.push('</tr>');
+    }
+    html.push('</tbody>');
+
+    html.push('</table>');
+
+    container.innerHTML = html.join('\n');
+}
+
+renderTable(container1, table);
+renderTable(container2, tableModified);
+
+
+// Synchronize MouseOver
+
+const cursor = new Highcharts.DataCursor();
```

Comment:
Is it going to be on Highcharts names as it is now or will it be part of the dashboards?